### ISSUE TITLE
offset is incorrect for -00:30 when parsed from ISO and  { setZone: true }

### ISSUE THREAD

**Issue by mceachen** [moment/luxon#580]
```js
const DateTime = require("luxon").DateTime
> DateTime.fromISO("2019-09-24T11:57:31.653-00:30", { setZone: true }).offset
30
> DateTime.fromISO("2019-09-24T11:57:31.653+00:30", { setZone: true }).offset
30
> DateTime.fromISO("2019-09-24T11:57:31.653-01:00", { setZone: true }).offset
-60
> DateTime.fromISO("2019-09-24T11:57:31.653+01:00", { setZone: true }).offset
60
```

Expected: `DateTime.fromISO("2019-09-24T11:57:31.653-00:30", { setZone: true }).offset` should be `-30`.

Note that all other offsets (-14:00 through +14:00, stepping every 30 minutes) seems to work correctly.

This is running Luxon v1.19.2, node v12.4.0.

Most importantly: thanks for Luxon!

**Comment by icambron** [comment]
Yup, looks like a bug.

**Comment by icambron** [comment]
Fixed in 9f92020, released in 1.19.3

**Comment by mceachen** [comment]
YOU ROCK
